frontend/src/views/WasteActionsPage/WasteActionPage.vue

```diff
@@ -102,6 +121,18 @@ export default {
         }
       })
     },
+    hasActionsDone() {
+      return this.wasteAction?.actions?.filter((action) => action.isDone).length > 0
+    },
+    actionsDone() {
+      return this.wasteAction.actions
+        .filter((action) => action.isDone)
+        .map((action) => {
```

Comment:
```suggestion
        .map((action) => ({ text: action.canteen.name }))
```
marche aussi je crois